### ISSUE TITLE
Apply now button behaviour

### ISSUE THREAD

**Issue by ldabiralai** [sky-uk/developers-sky-com#141]
As @andrewmerrell mentioned yesterday, we need to remove the form to submit details as we don't serve over https yet. 

Where should the apply now button point to?

**Comment by mubzymalone** [comment]
I assumed we would wait until we serve over https before going live and hence keep that section?

**Comment by ldabiralai** [comment]
I think the issue was that github pages doesn't support https.

I don't see why we should let this hold us back from going live. The temporary fix, in my opinion would be to remove it for now (or change where it goes) until we can find a solution to serving the site over https.

Thoughts @andrewmerrell ?

**Comment by andrewmerrell** [comment]
I suggest we remove the link for now and point people towards the careers website, not ideal but would unblock us.

Wondering if http://developers.theguardian.com/join-the-team.html is a good model (not ideal...) they link out to the telo portal, we could do the same!

**Comment by behaghel** [comment]
I agree, priority should be to go live. Apparently Telo portal is not good enough and one major goal of this project is to deliver a better experience so I would suggest we don't get tempted to link to it as a temporary move. It may prevent us from achieving as intended due to the usual fate of temporary hacks.
